src/html/erc20-form.html

```diff
@@ -114,8 +128,13 @@ <h3>Enter token address</h3>
     select.onclick = () => window.dom.show('erc20Modal')
 
     amount.onkeyup = function enableOrDisable (e) {
-      if (Number(e.target.value)) submit.disabled = false
+      const balance = Number(window.dom.find('erc20Balance').innerHTML)
+      const allowance = Number(window.dom.find('erc20Allowance').innerHTML)
+      const val = e.target.value
+      if (val && Number(val) <= balance && Number(val) > 0) submit.disabled = false
```

Comment:
might be worth extracting a helper for this `val = ; if (val && Number(val)...` stuff. Seems like it will be a common pattern beyond these two files.